editor/editor_properties_array_dict.cpp

```diff
@@ -517,6 +543,62 @@ void EditorPropertyArray::drop_data_fw(const Point2 &p_point, const Variant &p_d
 		emit_changed(get_edited_property(), array, "", false);
 		update_property();
 	}
+
+	if (drag_data.has("type") && String(drag_data["type"]) == "nodes") {
+		Array nodes = drag_data["nodes"];
+		Array nodes_paths = Array();
+		Node *base_node = get_base_node();
+		nodes_paths.resize(nodes.size());
+		for (size_t i = 0; i < nodes.size(); i++) {
+			Node *node = get_tree()->get_edited_scene_root()->get_node(nodes[i]);
+			NodePath path = node->get_path();
+
+			if (!base_node && Object::cast_to<RefCounted>(get_edited_object())) {
+				Node *to_node = get_node(node->get_path());
+				ERR_FAIL_NULL(to_node);
+				path = get_tree()->get_edited_scene_root()->get_path_to(to_node);
+			}
+
+			if (base_node) {
+				path = base_node->get_path().rel_path_to(node->get_path());
+			}
+
+			nodes_paths[i] = path;
+		}
+
+		emit_changed(get_edited_property(), nodes_paths, "", false);
+		update_property();
+	}
+}
+
+Node *EditorPropertyArray::get_base_node() {
```

Comment:
I pretty much copied this function from `EditorPropertyNodePath` in `editor_properties.cpp` because I could not access it. Tell me if there's a better way.